cfgov/unprocessed/css/molecules/submit-complaint-cta.less

```diff
@@ -0,0 +1,46 @@
+/* topdoc
+  name: Submit a Complaint Call to Action (CTA)
+  family: cfgov-molecules
+  patterns:
+    - name: Default example
+      markup: |
+        <div class="m-submit-complaint-cta">
+            <a href="#">Submit a Complaint</a>
+        </div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .m-submit-complaint-cta
+            a
+  tags:
+    - cfgov-molecules
+*/
+
+.m-submit-complaint-cta {
+    border-left: 1px solid @gray-50;
+    padding-left: unit( @grid_gutter-width / 2 / @base-font-size-px, em );
+    padding-top: unit( @grid_gutter-width / 3 / @base-font-size-px, em );
+    padding-bottom: unit( @grid_gutter-width / 3 / @base-font-size-px, em );
+
+    a {
+        .webfont-medium();
+        color: @pacific;
```

Comment:
Otherwise it looks like this:

<img width="224" alt="screen shot 2016-01-20 at 9 26 42 am" src="https://cloud.githubusercontent.com/assets/704760/12451525/fc43880c-bf57-11e5-9669-f7dda69b4f36.png">

Looks to be picking up the same color as the beta banner link.